AK/FloatingPoint.h

```diff
@@ -22,16 +22,16 @@ union FloatExtractor<f128> {
     static constexpr int exponent_bias = 16383;
     static constexpr int exponent_bits = 15;
     static constexpr unsigned exponent_max = 32767;
-    struct {
+    struct [[gnu::packed]] {
         unsigned __int128 mantissa : 112;
         unsigned exponent : 15;
         unsigned sign : 1;
```

Comment:
Shouldn't these also be `unsigned __int128`?

Reply:
Technically, but those are ifdef'd out on windows anyway.
I'll change these too.